Makefile.system

```diff
@@ -641,18 +641,22 @@ DYNAMIC_CORE += POWER8
 ifneq ($(C_COMPILER), GCC)
 DYNAMIC_CORE += POWER9
 DYNAMIC_CORE += POWER10
+CCOMMON_OPT += -DHAVE_P10_SUPPORT
 endif
 ifeq ($(C_COMPILER), GCC)
 ifeq ($(GCCVERSIONGT5), 1)
 DYNAMIC_CORE += POWER9
 else
 $(info, OpenBLAS: Your gcc version is too old to build the POWER9 kernels.)
 endif
-ifeq ($(GCCVERSIONGTEQ11), 1)
+LDVERSIONGTEQ35 := $(shell expr ld --version | head -1 | cut -f2 -d "." | cut -f1 -d "-" >= 35)
+ifeq ($(GCCVERSIONGTEQ11)$(LDVERSIONGTEQ35), 11)
```

Comment:
This if condition seems to not work even if both conditions are met.. checking with more options...

Reply:
This is due to missing backquote. 
LDVERSIONGTEQ35 := $(shell expr `ld --version | head -1 | cut -f2 -d "."  | cut -f1 -d "-"` \>= 35)

Reply:
Indeed, thanks. Somehow they got lost in a copy+paste between two remote shells.